docs/source/config/webui.rst

```diff
@@ -47,7 +47,7 @@ On a web client side, a file ``config.js`` in a root of the project which contai
 
 Multiple servers could be configured for user to pick from. To disconnect from the current server and return to login screen, pick 'Disconnect' from the drop down at the top right corner of the UI.
 
-On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com:8000`, your config should look like that:
+On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com`, your config should look like that:
```

Comment:
Guys, a note on language: `standalone` is used all over to separate two 'auth' modes. We also need a word to separate running under WSGI app server (with nginx) from running "standalone" (pardon my language) == with simple http server embedded in pecan.

This becomes especially obvious when you look at auth docs and config files.
Please come up with something. I did a try in the upcoming PR>